debian/assets/carlos_ctl/o19etl.py

```
@@ -0,0 +1,1401 @@
+    from . import o19roles
+    # every plain-client statement interpolates values with backslash
+    # escaping; a server mode that disables it (or ANSI_QUOTES) would turn
+    # a quoted clinic value into SQL. The ETL executor pins sql_mode='';
+    # the plain client runs under the server's own, so it is checked once
+    mode = plain("SELECT @@SESSION.sql_mode")[0][0].upper()
+    for flag in ("NO_BACKSLASH_ESCAPES", "ANSI_QUOTES"):
+        if flag in mode:
+            problems.append("the server's sql_mode carries {0}; the import "
+                            "quotes clinic values with backslash escapes "
+                            "and refuses to run under it — clear it for "
+                            "the import".format(flag))
```

Ruff (0.16.3)

[warning] Use `list.extend` to create a transformed list

(PERF401)